### ISSUE TITLE
Stacked bar charts could have 'Relative' toggle

### ISSUE THREAD

**Issue by bastianherre** [owid/owid-grapher#2051]
## Core problem

In some stacked bar charts it would be nice to switch between absolute and relative views, just like for stacked area charts. Examples are [Deaths in state-based conflicts by world region](https://ourworldindata.org/grapher/deaths-in-state-based-conflicts-by-world-region) and [State-based conflicts, World, 1946 to 2020](https://ourworldindata.org/grapher/the-number-of-active-state-based-conflicts?country=~OWID_WRL).

## Proposed solution

The solution would be to have a relative toggle for stacked bar charts, too.

## Alternatives

The alternative is leaving the Grapher as is, and using more stacked area charts. But stacked area charts sometimes don't look great with spiky data, as in the chart above.

## Context

Max encouraged me to bring this to your attention when he saw me using stacked areas where he preferred stacked bars.

Please let me know if you have any questions! Thank you!

**Comment by larsyencken** [comment]
Agreed that this is a nice feature to have, and hopefully not difficult to implement.

**Comment by marcelgerber** [comment]
Hi Bastian, this is now possible.
The option is hidden on all existing charts, but can be easily enabled in the Admin.

Note, though, that in relative mode, https://github.com/owid/owid-grapher/issues/1865 gets even worse: All years with a total of zero will be completely dropped (because the percentage cannot be computed for them), no matter what.
This is not an issue for your charts, thankfully.
